src/metatrain/utils/llpr.py

```diff
@@ -180,7 +180,16 @@ def forward(
                 requested_ensembles.append(name)
 
         for name in requested_ensembles:
-            ensemble_weights = getattr(self, name + "_weights")
+            # get the ensemble weights (getattr not supported by torchscript)
+            ensemble_weights = torch.tensor(0.0)
+            for buffer_name, buffer in self.named_buffers():
+                if buffer_name == name + "_weights":
+                    ensemble_weights = buffer
+            if len(ensemble_weights.shape) == 0:
+                # ensemble weights not found
+                raise ValueError(
```

Comment:
Can you try to cover the error message?

Reply:
Sure

Reply:
Actually it's untestable because it fails before (since the output is not registered in the model), so I'll take it out and replace with a comment